src/plugins/intel_npu/tests/unit/CMakeLists.txt

```diff
@@ -20,7 +20,7 @@ ov_add_test_target(
         NAME ${TARGET_NAME}
         ROOT ${CMAKE_CURRENT_SOURCE_DIR}
         ADDITIONAL_SOURCE_DIRS
-            ${OpenVINO_SOURCE_DIR}/src/plugins/intel_npu/src/plugin/npuw/
+            ${OpenVINO_SOURCE_DIR}/src/plugins/intel_npu/src/plugin/npuw/partitioning
```

Comment:
Doesn't it break other includes in other test files? I believe we explicitly use `npuw/` path